### PR TITLE
Serendipity: A new welcome app

### PR DESCRIPTION
**Pull request by thankyouverycool**
Inspired by nvella's wizard (#5322, #5534), here's another nostalgic window. 

serendipity

The last welcome app got axed for failing to uphold The Vision™ so the idea this time is to take the system and user seriously and provide interesting tips beyond beginner basics. The initial tips include some features and shortcuts that aren't immediately discoverable unless you've been lurking the commits. If you've been holding on to any dank keyboard combos or esoteric serenity knowledge, tips.txt is the place to confess!

### CODE REVIEW

**Comment by tomuta** [comment]
Ah yeah we really need this for all those hotkeys where the only way to find them is to read the code ;)

**Comment by tomuta** [comment]
How about a clippy clone? https://en.wikipedia.org/wiki/Office_Assistant

**Comment by thankyouverycool** [comment]
> How about a clippy clone?

his name is buggy buddy.
future-bar2

**Comment by tomuta** [comment]
> > How about a clippy clone?
> 
> his name is buggy buddy.
> future-bar2

Haha that's hilarious! Can I have a patch for this? lol

File: Base/home/anon/Documents/tips.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# SerenityOS Tips
+# Did you know...
+Pressing Ctrl+Alt+Space at an insertion caret pops up the emoji picker.
+Holding Super while scrolling the mouse wheel over a window adjusts its transparency.
+Browser has built-in ad blocking. Filter content by adding new domains to ~/.config/BrowserContentFiltering.txt
+Default file and protocol associations can be changed in ~/.config/LaunchServer.ini
+Text Editor has multiple viewing modes; edit and preview HTML and Markdown in real time.
+It can help to get a second pair of $ Eyes on a problem. Or fifty: $ Eyes -n 100
+Highlighted text in Terminal can be launched or right-clicked for more context.
+Focus can be cycled between windows by pressing and holding Super+Tab. Shift reverses the order.
```
Comment (alimpfard) [inline Base/home/anon/Documents/tips.txt:10]: WHAT?
How come I've never seen this...?

**Comment by awesomekling** [comment]
This is really neato! The only thing I dislike here is the awkward looking large font, but it's the best we have at that size right now :)

File: Base/home/anon/Documents/tips.txt (hunk 1 above)
Comment (awesomekling) [reply Base/home/anon/Documents/tips.txt:10]: Please open new bugs as new bugs instead. :)

**Comment by thankyouverycool** [comment]
> awkward looking large font

liza is an abomination, but she's ours. I cheated and kerned a banner for now^
banner

File: Base/home/anon/Documents/tips.txt (hunk 1 above)
Comment (alimpfard) [reply Base/home/anon/Documents/tips.txt:10]: Me actually reporting bugs? now that's one impossible thing turned possible.

**Pull request merged**